build.cake

```diff
@@ -1,5 +1,8 @@
-#tool "nuget:?package=GitReleaseNotes"
-#tool nuget:?package=GitVersion.CommandLine
+// Enable Cake aliases to execute .NET global tools
+#module nuget:?package=Cake.DotNetTool.Module&version=0.4.0
+
+#tool nuget:?package=GitReleaseNotes&version=0.7.1
+#tool dotnet:?package=GitVersion.Tool&version=5.1.2
```

Comment:
Cake adds warnings to the build output if specific versions are not used. The nice part is that it also speeds up the NuGet querying to compile on each run.

Reply:
@jnm2 we actually always recommend pinning to specific versions of all tools/addins/modules, we are just becoming a little bit more vocal about it 😄